### ISSUE TITLE
`cargo install` failed

### ISSUE THREAD

**Issue by coriolinus** [birkenfeld/fddf#1]
```shell
$ cargo install fddf
    Updating registry `https://github.com/rust-lang/crates.io-index`
 Downloading fddf v1.0.0
  Installing fddf v1.0.0
 Downloading num_cpus v1.5.1
 Downloading scoped-pool v1.0.0
 Downloading sha1 v0.2.0
 Downloading variance v0.1.3
 Downloading scopeguard v0.1.2
 Downloading atty v0.2.2
 Downloading term_size v0.3.0
   Compiling bitflags v0.8.2
   Compiling ansi_term v0.9.0
   Compiling unicode-segmentation v1.2.0
   Compiling libc v0.2.23
   Compiling crossbeam v0.2.10
   Compiling unicode-width v0.1.4
   Compiling scopeguard v0.1.2
   Compiling vec_map v0.8.0
   Compiling strsim v0.6.0
   Compiling variance v0.1.3
   Compiling same-file v0.1.3
   Compiling fnv v1.0.5
   Compiling sha1 v0.2.0
   Compiling walkdir v1.0.7
   Compiling atty v0.2.2
   Compiling term_size v0.3.0
   Compiling num_cpus v1.5.1
   Compiling clap v2.24.2
   Compiling scoped-pool v1.0.0
   Compiling fddf v1.0.0
error: cannot find macro `eprintln!` in this scope
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/fddf-1.0.0/src/main.rs:19:9
   |
19 |         eprintln!("Hashing {}...", path.display());
   |         ^^^^^^^^
   |
   = help: did you mean `println!`?

error: cannot find macro `eprintln!` in this scope
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/fddf-1.0.0/src/main.rs:31:13
   |
31 |             eprintln!("Error opening file {}: {}", path.display(), e);
   |             ^^^^^^^^
   |
   = help: did you mean `println!`?

error: aborting due to 2 previous errors

error: failed to compile `fddf v1.0.0`, intermediate artifacts can be found at `/tmp/cargo-install.wemz5jNXsnIn`

Caused by:
  Could not compile `fddf`.

To learn more, run the command again with --verbose.
```

```shell
$ rustc --version ; cargo --version
rustc 1.17.0 (56124baa9 2017-04-24)
cargo 0.18.0 (fe7b0cdcf 2017-04-24)
```

**Comment by birkenfeld** [comment]
Whoops, nightly snuck in. Will fix it to run on stable too.

**Comment by birkenfeld** [comment]
Here you go, fixed in 1.0.1 - thanks for the report!
